OutOfSchool/OutOfSchool.WebApi/Services/Database/WorkshopService.cs

```diff
@@ -547,6 +547,20 @@ public async Task<Guid> GetWorkshopProviderOwnerIdAsync(Guid workshopId)
             .ConfigureAwait(false)).ProviderId;
     }
 
+    public async Task<List<ShortEntityDto>> GetWorkshopListByProviderAdminId(Guid providerId)
+    {
+        logger.LogDebug($"Getting Workshop (Id, Title) by organization started. Looking ProviderAdminId = {providerId}.");
+
+        var workshops = await workshopRepository.Get(
+                where: x => x.ProviderAdmins.Any(p => p.ProviderId == providerId))
+            .ToListAsync()
+            .ConfigureAwait(false);
+
+        var result = mapper.Map<List<ShortEntityDto>>(workshops).OrderBy(entity => entity.Title).ToList();
```

Comment:
please move order by to the repository Get method (using one of it's sorting arguments), if by any change we some time need pagination here - will be less time to refzctor :)